spec/defines/collectd_plugin_network_server_spec.rb

```diff
@@ -41,6 +41,18 @@
           is_expected.to contain_file("#{options[:plugin_conf_dir]}/network-server-node1.conf").with(ensure: 'absent')
         end
       end
+
+      context 'with specifique title' do
+        let(:title) { 'eatapples' }
+        let :params do
+          { servers: '1.1.1.1' }
```

Comment:
`1.1.1.1` belongs to APNIC-LABS. Better choose an RFC1918 IP address such as 10.0.0.1